### DISCUSSION TITLE
Meteorological data: Download data - execution fails

### DISCUSSION THREAD

**Discussion by bluntti** [UMEP-dev/UMEP#601] [Q&A]
Hello,

Whenever trying to run this tool I get the following error:

raise Exception('Missing/incomplete configuration file: %s' % (dotrc))
Exception: Missing/incomplete configuration file: C:\Users\my_user/.cdsapirc

However, I added a .txt file with the required KEY value as instructed and to this exact folder location highlighted.

Anyone can help me move forward?

I am aware that I could get the weather data from shinyweatherdata.com but it will be exported as a .csv file. Would that work with the following UMEP tools?

**Comment by biglimp** [comment]
Try to create a new KEY-file by using the instructions [here ](https://cds.climate.copernicus.eu/api-how-to). 

Regarding shinyweatherdata.com, the easiest way is to download the data as .epw-files and then just run that file through out [MetdataProcessor](https://umep-docs.readthedocs.io/en/latest/pre-processor/Meteorological%20Data%20MetPreprocessor.html).
